spec/requestor/district_attorney_spec.rb

```diff
@@ -0,0 +1,10 @@
+RSpec.describe OscnScraper::Requestor::DistrictAttorney do
+    describe '#perform' do
+  it 'gets the html' do
+    test = OscnScraper::Requestor::DistrictAttorney.new
+    data = test.perform
+
+    expect(data.class).to eq(HTTParty::Response)
+  end
+end
+  end
```

Comment:
I think we need to figure out how to re-enable the block alignment cop to save us from file spacing like this 😆